apps/wallet/src/ui/app/pages/home/tokens/TokensDetails.tsx

```diff
@@ -269,7 +269,7 @@ export function TokenDetails({ coinType }: TokenDetailsProps) {
                     ) : (
                         <div className="flex w-full flex-grow flex-col gap-md">
                             <div
-                                className={`flex w-full flex-grow flex-col items-center gap-xs rounded-2xl ${!accountHasIota ? 'justify-between' : ''}`}
```

Comment:
I think vested staking and migration shouldn't be pushed down when there's no coins 
![image](https://github.com/user-attachments/assets/8b9de6f9-5cc3-4022-86a3-d763318459bd)